java-client/example-utilities/build.gradle

```diff
@@ -5,7 +5,4 @@ plugins {
 dependencies {
     api 'info.picocli:picocli:4.6.1'
     api project(':java-client-session')
-
-    Classpaths.inheritGrpcPlatform(project, 'api')
-    api 'io.grpc:grpc-core'
```

Comment:
as an API dependency, this should still be here because _this_ project includes grpc-core in its API - if upstream changed to be implementation, that would break this project's _downstream consumers_, we might not even notice here...